### PR TITLE
Check multiple times for windows guests drivers after conversion

### PR DESCRIPTION
**Pull request by chloerh**
In case got in-accurate test result if drivers not fully installed
yet.
Sleep 60 seconds during each check.
Log the failure if still couldn't find driver after checks run out.

### CODE REVIEW

File: provider/v2v_vmcheck_helper.py [hunk 1]
```diff
@@ -283,13 +282,28 @@ def check_windows_vm(self):
         if self.os_version in ['win8', 'win8.1', 'win10', 'win2012', 'win2012r2']:
             expect_adapter = 'Basic Display Driver'
         expect_drivers.append(expect_adapter)
-        for driver in expect_drivers:
-            if driver in win_dirves:
-                logging.info("Find driver: %s", driver)
-                logging.info("PASS")
+        check_drivers = expect_drivers[:]
+        for check_times in range(5):
```
Comment (will-Do) [inline provider/v2v_vmcheck_helper.py]: could be use 'wait_for'

**Comment by balamuruhans** [comment]
please address the comments

File: provider/v2v_vmcheck_helper.py (hunk 1 above)
Comment (chloerh) [inline provider/v2v_vmcheck_helper.py]: There is this list of drivers to be checked. A driver will be removed from the list if it is found in guest during each loop of driver checking operation. The 'wait_for' function will need a function and check the return value, which in this case should be 'expect_drivers' is empty. I don't think it's more clear of the code if use the 'wait_for' function. I believe this way is more simple and clear.

**Comment by chloerh** [comment]
Recent test result

kvm.windows.win10.i386.qcow2_f.NFS | 16 分 | Passed
kvm.windows.win2008.i386.raw_f.NFS | 16 分 | Passed
kvm.windows.win2016.x86_64.qcow2_f.NFS | 21 分 | Fixed
kvm.windows.win8.x86_64.raw_f.NFS | 18 分 | Passed
kvm.windows.win8_1.i386.qcow2_f.NFS | 18 分 | Passed

File: provider/v2v_vmcheck_helper.py [hunk 2]
```diff
@@ -283,13 +282,28 @@ def check_windows_vm(self):
...
+            logging.info('Check drivers for the %dth time', check_times + 1)
+            win_dirves = self.checker.get_driver_info()
+            for driver in expect_drivers:
+                if driver in win_dirves:
+                    logging.info("Find driver: %s", driver)
```
Comment (balamuruhans) [inline provider/v2v_vmcheck_helper.py]: `logging.info("Driver %s found", driver)`

File: provider/v2v_vmcheck_helper.py [hunk 3]
```diff
@@ -283,13 +282,28 @@ def check_windows_vm(self):
...
+                    logging.info("PASS")
```
Comment (balamuruhans) [inline provider/v2v_vmcheck_helper.py]: I dont think this is required, it might be used during dev/debug

File: provider/v2v_vmcheck_helper.py [hunk 4]
```diff
@@ -283,13 +282,28 @@ def check_windows_vm(self):
...
+                    check_drivers.remove(driver)
+                else:
+                    err_msg = "Not find driver: %s" % driver
```
Comment (balamuruhans) [inline provider/v2v_vmcheck_helper.py]: `err_msg = "Driver %s not found" % driver`

**Comment by chloerh** [comment]
@balamuruhans Thanks for the review, all updated

**Pull request merged**